install.sh

```diff
@@ -184,6 +184,9 @@ for i in $(seq 0 $((${#urls[@]} - 1))); do
   update_device_json "${name}" "${version}"
 done
 
+# workaround https://github.com/skycocker/chromebrew/issues/3305
+sudo ldconfig > /dev/null 2> /dev/null || true
```

Comment:
I'm not following how this solves anything.

Reply:
On a bare bones installation, the ld cache does include files in /usr/local/lib64 , because chromebrew hasnt been installed yet.  After installing libiconv, its libraries are not usable by `git` until the ld cache has been updated.

If you enable https://travis-ci.org/uberhacker/chromebrew , and copy my .travis.yml onto a branch and commit and push it to GitHub, you can have your own build on Travis without this change to install.sh , and you can see the problem, and confirm this change is required.

But for your convienience , here is a build without this line:

https://travis-ci.org/jayvdb/chromebrew/builds/558512966#L503

```console
$ docker run -it -v $PWD:/travis brunux/chromeos /bin/sh -c "cd /travis; bash install.sh"
Chromebrew should not be installed or run as root.
Creating new device.json...
Downloading gcc8...
######################################################################## 100.0%
Verifying gcc8...
gcc8-8.3.0-chromeos-x86_64.tar.xz: OK
Extracting gcc8 (this may take a while)...
Installing gcc8 (this may take a while)...
Adding new information on gcc8 to device.json...
Downloading ruby...
######################################################################## 100.0%
Verifying ruby...
ruby-2.5.3-chromeos-x86_64.tar.xz: OK
Extracting ruby (this may take a while)...
Installing ruby (this may take a while)...
Adding new information on ruby to device.json...
Downloading libiconv...
######################################################################## 100.0%
Verifying libiconv...
libiconv-1.15-1-chromeos-x86_64.tar.xz: OK
Extracting libiconv (this may take a while)...
Installing libiconv (this may take a while)...
Adding new information on libiconv to device.json...
Downloading git...
######################################################################## 100.0%
Verifying git...
git-2.22.0-chromeos-x86_64.tar.xz: OK
Extracting git (this may take a while)...
Installing git (this may take a while)...
Adding new information on git to device.json...
Downloading libssh2...
######################################################################## 100.0%
Verifying libssh2...
libssh2-1.8.1-chromeos-x86_64.tar.xz: OK
Extracting libssh2 (this may take a while)...
Installing libssh2 (this may take a while)...
Adding new information on libssh2 to device.json...
git: error while loading shared libraries: libcharset.so.1: cannot open shared object file: No such file or directory
```

Reply:
This message is intriguing:
```
Chromebrew should not be installed or run as root.
```
How are you getting past this which should produce an error and abort the script?  Is there a way to create a new `chronos` user in docker and run under that user instead?  I'm guessing this is an environment related issue.

Reply:
Disregard that .. that doesnt happen in the submitted version of .travis.yml in this PR.

> Is there a way to create a new chronos user in docker and run under that user instead? I'm guessing this is an environment related issue.

This PR does that.